js_modules/dagster-ui/packages/ui-core/src/assets/AssetsCatalogTable.tsx

```diff
@@ -10,190 +10,25 @@ import {AssetTable} from './AssetTable';
 import {ASSET_TABLE_DEFINITION_FRAGMENT, ASSET_TABLE_FRAGMENT} from './AssetTableFragment';
 import {AssetsEmptyState} from './AssetsEmptyState';
 import {AssetTableFragment} from './types/AssetTableFragment.types';
-import {
-  AssetCatalogGroupTableNodeFragment,
-  AssetCatalogGroupTableQuery,
-  AssetCatalogGroupTableQueryVariables,
-  AssetCatalogTableQuery,
-  AssetCatalogTableQueryVariables,
-  AssetCatalogTableQueryVersion,
-} from './types/AssetsCatalogTable.types';
+import {useAllAssets} from './useAllAssets';
 import {AssetViewType, useAssetView} from './useAssetView';
-import {gql, useApolloClient} from '../apollo-client';
-import {AppContext} from '../app/AppContext';
+import {gql} from '../apollo-client';
 import {PYTHON_ERROR_FRAGMENT} from '../app/PythonErrorFragment';
 import {PythonErrorInfo} from '../app/PythonErrorInfo';
 import {FIFTEEN_SECONDS, useRefreshAtInterval} from '../app/QueryRefresh';
 import {currentPageAtom} from '../app/analytics';
-import {PythonErrorFragment} from '../app/types/PythonErrorFragment.types';
 import {tokenForAssetKey} from '../asset-graph/Utils';
 import {useAssetSelectionInput} from '../asset-selection/input/useAssetSelectionInput';
-import {getAssetsByKey} from '../asset-selection/util';
 import {AssetGroupSelector} from '../graphql/types';
-import {useUpdatingRef} from '../hooks/useUpdatingRef';
 import {useBlockTraceUntilTrue} from '../performance/TraceContext';
-import {fetchPaginatedData} from '../runs/fetchPaginatedBucketData';
-import {getCacheManager} from '../search/useIndexedDBCachedQuery';
 import {SyntaxError} from '../selection/CustomErrorListener';
 import {LoadingSpinner} from '../ui/Loading';
 
-type Asset = AssetTableFragment;
+export {useAllAssets} from './useAllAssets';
```

Comment:
This is a little odd, why re-export here?

Reply:
Didn't want to update a lot of callsites in this PR (and internal) but I can do that in a follow up.

Reply:
Ahh nice!